exo-deploy/package.json

```diff
@@ -4,20 +4,20 @@
   "author": "Hugo Ho",
   "dependencies": {
     "async": "2.1.4",
-    "aws-sdk": "2.7.10",
+    "aws-sdk": "2.7.27",
     "chalk": "1.1.3",
     "dashify": "0.2.2",
-    "fs-extra": "1.0.0",
-    "handlebars": "4.0.5",
-    "inquirer": "1.2.3",
+    "fs-extra": "2.0.0",
+    "handlebars": "4.0.6",
+    "inquirer": "3.0.1",
```

Comment:
This is a bump of 2 major versions. I know the jump from 1 to 2 had some breaking changes. Are you sure this works still?